raft/raft_test.go

```diff
@@ -1225,6 +1225,126 @@ func TestLeaderStepdownWhenQuorumLost(t *testing.T) {
 	}
 }
 
+func TestLeaderSupersedingWithoutCheckQuorum(t *testing.T) {
+	a := newTestRaft(1, []uint64{1, 2, 3}, 10, 1, NewMemoryStorage())
+	b := newTestRaft(2, []uint64{1, 2, 3}, 10, 1, NewMemoryStorage())
+	c := newTestRaft(3, []uint64{1, 2, 3}, 10, 1, NewMemoryStorage())
+
+	nt := newNetwork(a, b, c)
+	nt.send(pb.Message{From: 1, To: 1, Type: pb.MsgHup})
+	if c.state != StateFollower {
```

Comment:
actually we can remove this entire test since it is already covered by other tests.